examples/backends/vllm/launch/disagg_multimodal_p_d.sh

```
@@ -10,12 +10,14 @@
 # Trade-off: prefill does vision encoding internally (no dedicated encoder),
 # which uses more GPU memory on the prefill worker.
 set -e
-trap 'echo Cleaning up...; kill 0' EXIT
 
 SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
 source "$SCRIPT_DIR/../../../common/gpu_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/gpu_utils.sh was not specified as input (see shellcheck -x).

(SC1091)